packages/@webex/internal-plugin-metrics/test/unit/spec/call-diagnostic/call-diagnostic-metrics.ts

```diff
@@ -752,7 +755,9 @@ describe('internal-plugin-metrics', () => {
         assert.deepEqual(webexLoggerLogCalls[2].args, [
           'call-diagnostic-events -> ',
           'CallDiagnosticMetrics: @createClientEventObjectInMeeting => collected browser data',
-          '{"error":"unable to access window.navigator.userAgent"}',
+          `{"_ua":${JSON.stringify(
+            userAgent
+          )},"parsedResult":{"browser":{"name":"webex-js-sdk","version":"test-webex-version client=Cantina;"},"os":{},"platform":{},"engine":{}}}`,
```

Comment:
This change is because of `jsdom-global` which adds the window object and now we can get the `userAgent`.